src/session.cpp

```diff
@@ -328,7 +335,7 @@ session_t::memory_pressure() const {
 
 std::string
 session_t::name() const {
```

Comment:
I've been thinking about it for some time, actually. Does BH have some special handling magic for optional attributes?